src/dispatchers/posts.ts

```diff
@@ -20,10 +22,15 @@ export const deletePost = (id: number) => async (dispatch: AppDispatch) => {
   dispatch(unsetPost(id));
 };
 
-export const getPosts = (params?: GetPostsParams) => async (dispatch: AppDispatch) => {
-  const responseData = await _getPosts(params);
+export const resetPosts = () => (dispatch: AppDispatch) => {
+  dispatch(_resetPosts());
+};
 
-  for (const post of responseData) {
+export const getPosts = (params?: GetPostsParams) => async (dispatch: AppDispatch) => {
+  dispatch(startLoading());
+  const nextURL = getNextUrlFromState(store.getState().posts);
```

Comment:
Not sure if this is the right place to trigger loading and getting `next url` from state